extensions/vscode/webviews/homeView/src/components/TextStringWithAnchor.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <span v-if="activeOption" :class="class">
+    {{ messageParts[0] }}
+    <a class="webview-link" role="button" @click="onClick()">
+      {{ activeOption?.anchorStr }}
+    </a>
+    {{ messageParts[1] }}
+  </span>
+  <span v-else :class="class">
+    {{ message }}
+  </span>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+
+defineOptions({
+  inheritAttrs: false,
+});
+
+const emit = defineEmits<{
+  // <eventName>: <expected arguments>
+  click: [splitOptionId: number]; // named tuple syntax
+}>();
+
+interface SplitOption {
+  detectionStr: string;
+  anchorStr: string;
+  id: number;
+}
+
+interface Props {
+  message: string;
+  splitOptions: SplitOption[];
+  class?: string;
```

Comment:
Since I have multiple root elements within the component, I was doing this explicitly... but looks like I can use `$attrs.class` to go ahead and address that issue in a better way. OK, I will remove that prop.